containers/workload.tfaws.w

```diff
@@ -68,7 +68,7 @@ pub class Workload_tfaws impl api.IWorkload {
         );
     
         deployment.addContainer(
-          image: "{{ .Values.image }}",
+          image: "\{\{ .Values.image }}",
```

Comment:
Need to escape just the opening `{` and not the closing `}`?

Reply:
yes ([playground](https://www.winglang.io/play/?code=dABlAHMAdAAgACIAZQBzAGMAYQBwAGUAIgAgAHsACgAgACAAbABvAGcAKAAiAGgAZQBsAGwAbwAgAFwAewB3AG8AcgBsAGQAfQAiACkAOwAKAH0A))